src/main/cpp/Robot.cpp

```diff
@@ -121,7 +121,7 @@ void Robot::RobotPeriodic()
         feedback->UpdateFeedback();
     }
 
-    auto pigeon = RobotConfigMgr::GetInstance()->GetCurrentConfig()->GetPigeon(CanSensorUsage::CANSENSOR_USAGE::PIGEON_ROBOT_CENTER);
+    auto pigeon = RobotConfigMgr::GetInstance()->GetCurrentConfig()->GetPigeon(RobotElementNames::PIGEON_USAGE::PIGEON_ROBOT_CENTER);
```

Comment:
I think we can remove this logging from Robot.cpp (swerve chassis already does this).

The cleaner we can keep Robot.cpp the better.

Reply:
removed logging and committed